unity/unity-embed-host/CoreCLRHost.cs

```diff
@@ -607,6 +608,42 @@ public static IntPtr unity_method_get_function_pointer([NativeCallbackType("Mono
         return t.IsValueType;
     }
 
+    private static ConcurrentDictionary<IntPtr, bool> s_isBlittableCache = new ConcurrentDictionary<IntPtr, bool>();
```

Comment:
Note we will need to clear this Dictionary of any types when assembly unload is supported. We may have another cache or two as well that needs handled in these cases.

cc @simonferquel @alexey-zakharov @unity-christianb

Reply:
Thanks for the info, and thanks @alexey-zakharov for the extra explanation offline :)